app/presenters/schools/on_boarding/candidate_requirements_selection_presenter.rb

```diff
@@ -14,13 +13,7 @@ def initialize(attributes)
       def to_s
         reqs = requirements
 
-        if reqs.empty?
-          'None'
-        else
-          content_tag('ul', class: 'govuk-list') do
-            safe_join(reqs.map { |req| tag.li(req) })
-          end
-        end
+        reqs.empty? ? 'None' : reqs.join("\n")
```

Comment:
I'd suggest that this just leaves requirements blank if there aren't any - display of blank requirements can be handled by the view helpers - no strong feelings here though